lib/membrane/rtp/rtx_parser.ex

```diff
@@ -0,0 +1,64 @@
+defmodule Membrane.RTP.RtxParser do
```

Comment:
I would consider using `RTX`, I think that most people write rtx using capital letters